e2e/rbd.go

```diff
@@ -366,6 +366,59 @@ var _ = Describe("RBD", func() {
 					}
 				})
 			}
+			By("validate RBD migration+static Block PVC Deletion", func() {
+				// create monitors hash by fetching monitors from the cluster.
+				mons, err := getMons(rookNamespace, c)
+				if err != nil {
+					e2elog.Failf("failed to get monitors %v", err)
+				}
+				mon := strings.Join(mons, ",")
+				inClusterID := getMonsHash(mon)
+				customInfo := map[string]string{
+					"clusterID-1": "subvolgrp1",
+					// the monitors hash entry in the configmap.
+					inClusterID: "subvolgrp2",
+				}
```

Comment:
IMO subvolgrp doesn't make sense for  RBD. please change this to something else

Reply:
Done.